docs/ember/helper-and-modifier-signatures.md

```diff
@@ -0,0 +1,151 @@
+Like components, helpers and modifiers can also have their behavior in templates described by a `Signature` type.
+
+## Helpers
+
+A helper signature has two keys: `Args` and `Return`. The `Args` key is further broken down into `Named` and
+`Positional` elements, representing respectively the expected types of the named and positional arguments your helper
+expects to receive.
+
+{% tabs %}
+{% tab title="Class-Based" %}
+
+```typescript
+import Helper from '@ember/component/helper';
+
+type Positional = Array<number>;
+type Named = { andThenMultiplyBy?: number };
+
+export interface AddSignature {
```

Comment:
Since signatures don't actually _need_ to be exported, is there any value in doing so?

Reply:
People sometimes like to do type gymnastics with them, and reverse engineering them from the component type itself generally makes TS fall over. Given the number of times I've had to point people to just exporting the signature type instead of trying to extract it, it seemed reasonable to do it by default.